src/Confluent.Kafka/Config_gen.cs

```diff
@@ -659,23 +723,23 @@ public class ClientConfig : Config
         public string SaslKerberosPrincipal { get { return Get("sasl.kerberos.principal"); } set { this.SetObject("sasl.kerberos.principal", value); } }
 
         /// <summary>
-        ///     Full kerberos kinit command string, %{config.prop.name} is replaced by corresponding config object value, %{broker.name} returns the broker's hostname.
+        ///     Shell command to refresh or acquire the client's Kerberos ticket. This command is executed on client creation and every sasl.kerberos.min.time.before.relogin (0=disable). %{config.prop.name} is replaced by corresponding config object value.
```

Comment:
This stuff was released in v1.1.0.
Maybe update your RELEASE.md guidelines to include the re-gen step so it is not forgotten in the future.

Reply:
can't update, don't have

Reply:
Highly recommend we make one, for when we need to make a release when you're in Mexico.

Reply:
for additional context: the config regen never made it into 1.1 because of the dependency on https://github.com/edenhill/librdkafka/pull/2416 (which i didn't pick up on before the 1.1 librdkafka release)